kedro-datasets/CONTRIBUTING.md

```diff
@@ -66,14 +102,19 @@ We use a branching model that helps us keep track of branches in a logical, cons
 | `fix`            | Non-breaking change which fixes an issue                                    |
 | `tests`          | Changes to project unit (`tests/`) and / or integration (`features/`) tests |
 
-## Plugin contribution process
+## Dataset contribution process
 
  1. Fork the project
  2. Develop your contribution in a new branch.
- 3. Make sure all your commits are signed off by using `-s` flag with `git commit`.
- 4. Open a PR against the `main` branch and sure that the PR title follows the [Conventional Commits specs](https://www.conventionalcommits.org/en/v1.0.0/) with the scope `(datasets)`.
- 5. Make sure the CI builds are green (have a look at the section [Running checks locally](#running-checks-locally) below)
- 6. Update the PR according to the reviewer's comments
+ 3. Add your dataset to the appropriate location:
```

Comment:
What's not clear about this workflow is how first-time contributors can know ahead of time whether the TSC will be willing to maintain the dataset, and therefore where to contribute it.

Reply:
You're totally right. Should we just say they open it as `experimental` and then the TSC does a vote on whether we want to maintain it or not? Which is basically a vote for "immediate graduation".